### PR TITLE
First round of client library improvements

### PR DESCRIPTION
**Pull request by josephschorr**

### CODE REVIEW

File: examples/v1alpha1/example.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const client = new authzed.Client("mytokenhere");
+
+const writeSchemaRequest = new authzed.WriteSchemaRequest();
+writeSchemaRequest.setSchema(`
+definition test/user {}
+
+definition test/resource {
+relation viewer: test/user
+permission view = viewer
+}
```
Comment (jzelinskie) [inline examples/v1alpha1/example.js:12]: can we fix the indentation here?

File: run-and-test.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+docker stop zed-testserver && docker rm zed-testserver
+docker run -d -p 50051:50051 --name=zed-testserver quay.io/authzed/zed-testserver:latest run
+sleep 1
+jasmine --config=jasmine.json
+docker stop zed-testserver
+docker rm zed-testserver
```
Comment (jzelinskie) [inline run-and-test.sh:6]: can we move this into a scripts directory?

File: src/authzedapi/authzed/api/v0/Namespace_serviceServiceClientPb.ts [hunk 3]
```diff
@@ -0,0 +1,119 @@
+/**
```
Comment (jzelinskie) [inline src/authzedapi/authzed/api/v0/Namespace_serviceServiceClientPb.ts:1]: can we add a .gitattributes entry to mark these as generated?

File: src/authzedapi/authzed/api/v0/acl_service.pb.validate.go [hunk 4]
```diff
@@ -0,0 +1,1273 @@
+// Code generated by protoc-gen-validate. DO NOT EDIT.
+// source: authzed/api/v0/acl_service.proto
```
Comment (jzelinskie) [inline src/authzedapi/authzed/api/v0/acl_service.pb.validate.go:2]: i think you accidentally generated go

File: src/authzedapi/authzed/api/v0/developer.pb.validate.go [hunk 5]
```diff
@@ -0,0 +1,1077 @@
+// Code generated by protoc-gen-validate. DO NOT EDIT.
```
Comment (jzelinskie) [inline src/authzedapi/authzed/api/v0/developer.pb.validate.go:1]: more go

**Review by jzelinskie — CHANGES_REQUESTED** [review]

File: src/authzedapi/authzed/api/v0/Namespace_serviceServiceClientPb.ts (hunk 3 above)
Comment (josephschorr) [reply src/authzedapi/authzed/api/v0/Namespace_serviceServiceClientPb.ts:1]: Done

File: run-and-test.sh (hunk 2 above)
Comment (josephschorr) [reply run-and-test.sh]: Done

File: examples/v1alpha1/example.js (hunk 1 above)
Comment (josephschorr) [reply examples/v1alpha1/example.js]: Fixed

**Pull request merged**